### ISSUE TITLE
Ellipsis for truncating story card text is webkit only

### ISSUE THREAD

**Issue by kookster** [PRX/publish.prx.org#58]

**Comment by sandikbarr** [comment]
The way I've seen we handle this elsewhere is by actually truncating the text. My concern with that approach is that it's not adaptable to different layouts like browser overflow is and we end up guessing how much text will fit in the space for the default size. Here is one approach to a similar problem I've used before: http://omahagirlswhocode.com/volunteers.html/https://github.com/OmahaGirlsWhoCode/OmahaGirlsWhoCode/blob/master/css/photo-grid.css The problem I ran into with that approach is not having a solution for when the text does not overflow. Any better ideas?
